common/src/main/java/com/wynntils/functions/LootrunFunctions.java

```diff
@@ -150,6 +150,32 @@ public FunctionArguments.Builder getArgumentsBuilder() {
         }
     }
 
+    public static class LootrunTimeFunction extends Function<Integer> {
+        @Override
+        public Integer getValue(FunctionArguments arguments) {
+            return Models.Lootrun.getTimer(arguments.getArgument("requiredTime").getBooleanValue());
+        }
+
+        @Override
+        public FunctionArguments.Builder getArgumentsBuilder() {
+            return new FunctionArguments.OptionalArgumentBuilder(
+                    List.of(new FunctionArguments.Argument<>("requiredTime", Boolean.class, false)));
```

Comment:
```suggestion
                    List.of(new FunctionArguments.Argument<>("timeLeftForTask", Boolean.class, false)));
```
Perhaps this is a better name. Fix the function and i18n aswell